lib/appenders/multiprocess.js

```diff
@@ -128,6 +128,7 @@ function workerAppender(config) {
     socket.on('timeout', socket.end.bind(socket));
     // don't bother listening for 'error', 'close' gets called after that anyway
     socket.on('close', createSocket);
+    socket.on('error', () => {});
```

Comment:
Might want to follow #1089 and flush the buffer on `'error'` before it calls `'close'`.

```javascript
    socket.on('error', (e) => {
      debug('connection error', e);
      canWrite = false;
      emptyBuffer();
    })
```